type-definitions/almin.js.flow

```diff
@@ -45,15 +60,15 @@ declare class Dispatcher extends events$EventEmitter {
     static isDispatcher(maybeDispatcher: mixed): boolean;
 
     onDispatch(payloadHandler: (payload: DispatcherPayload) => mixed): Function;
-    dispatch(payload: DispatcherPayload): void;
+    dispatch(payload: DispatcherPayload|Payload): void;
     pipe(toDispatcher: Dispatcher): Function;
 }
 
 declare class Store extends Dispatcher {
     static isStore(maybeStore: mixed): boolean;
 
     getState(prevState: Object): Object;
-    onChange(callback: Function): Function;
+    onChange(stores: Array<Store>): Function;
```

Comment:
It is a breaking change.
#81 https://github.com/almin/almin/pull/80#discussion_r101957371